test/testing_test.rb

```diff
@@ -213,11 +213,32 @@ def plan_input
         end
       end
 
-      it "is able to handle when events ===" do
+      it "is able to handle when events" do
```

Comment:
Apparently, Ruby is not as smart as it looks like, and can't tell what  I want to do just from the string description. I forgot to update the plan I wanted to actually test, and it turned out I had some issues with the event handling in the in thread executor, that should now be solved